### PR TITLE
PRESIDECMS-2284 Admin view record property no value

### PR DESCRIPTION
**Pull request by teonator**

### CODE REVIEW

File: system/handlers/admin/DataHelpers.cfc [hunk 1]
```diff
@@ -153,15 +158,19 @@ component extends="preside.system.base.adminHandler" {
 		var baseLink      = event.buildadminLink( objectName=relatedObject, recordId="{recordId}" );
 		var list          = [];
 
-		for( var record in records ) {
-			if ( baseLink.len() ) {
-				list.append( '<a href="#( baseLink.replace( '{recordId}', record.id ))#">#record.label#</a>' );
+		for ( var record in records ) {
+			var recordLabel = record.label;
+
+			recordLabel = _renderyNoValue( objectName=objectName, propertyName=propertyName, propertyValue=recordLabel );
```
Comment (DominicWatson) [inline system/handlers/admin/DataHelpers.cfc]: Am not sure why `_renderyNoValue()` is required here? - we know that we have a list of records, so there should always be something to render?

File: system/handlers/admin/DataHelpers.cfc [hunk 2]
```diff
@@ -281,4 +290,21 @@ component extends="preside.system.base.adminHandler" {
 		}
 	}
 
+	private string function _renderyNoValue(
```
Comment (DominicWatson) [inline system/handlers/admin/DataHelpers.cfc]: Should this be `_renderNoValue()` rather than `_renderyNoValue()`?

**Review by DominicWatson — CHANGES_REQUESTED** [review]
Nice one, looking good @teonator . Could you check the review comments and update?

File: system/handlers/admin/DataHelpers.cfc (hunk 1 above)
Comment (teonator) [reply system/handlers/admin/DataHelpers.cfc]: Removed.

File: system/handlers/admin/DataHelpers.cfc (hunk 2 above)
Comment (teonator) [reply system/handlers/admin/DataHelpers.cfc]: Fixed.

**Comment by DominicWatson** [comment]
Fab, merging into upcoming release for QA

**Pull request merged**